svir/dialogs/load_output_as_layer_dialog.py

```diff
@@ -404,18 +404,34 @@ def style_maps(self):
         symbol.symbolLayer(0).setOutlineStyle(Qt.PenStyle(Qt.NoPen))
 
         style = get_style(self.layer, self.iface.messageBar())
-        ramp = QgsVectorGradientColorRampV2(
-            style['color_from'], style['color_to'])
+        default_qgs_style = QgsStyleV2().defaultStyle()
+        default_color_ramp_names = default_qgs_style.colorRampNames()
+        ramp_type_idx = default_color_ramp_names.index('Spectral')
+        ramp = default_qgs_style.colorRamp(
+            default_color_ramp_names[ramp_type_idx])
+        inverted = True
+        mode = style['mode']
+        # in some cases, we override the user-specified setting, and use
+        # instead a setting that was required by scientists
+        if self.output_type in ('dmg_by_asset', 'losses_by_asset'):
+            mode = QgsGraduatedSymbolRendererV2.Jenks  # jenks = natural breaks
+            ramp_type_idx = default_color_ramp_names.index('Reds')
+            ramp = default_qgs_style.colorRamp(
+                default_color_ramp_names[ramp_type_idx])
+            inverted = False
+        elif self.output_type in ('hmaps', 'gmf_data'):
+            mode = QgsGraduatedSymbolRendererV2.EqualInterval
```

Comment:
LGTM but let's use the original default with user's colors for cases not evaluated by ifs